web/src/features/panels/ranking-panel/RankingPanel.tsx

```diff
@@ -55,16 +54,16 @@ export default function RankingPanel(): ReactElement {
   );
 
   return (
-    <div className="flex h-[calc(100vh-236px)] w-full flex-col py-3 pl-4 pr-1 ">
+    <div className="flex h-[calc(100vh-220px)] w-full flex-col py-3 pl-4 pr-1 ">
       <Helmet prioritizeSeoTags>
         <title>{t('misc.maintitle') + metaTitleSuffix}</title>
         <link rel="canonical" href={canonicalUrl} />
       </Helmet>
 
-      <div className="flex flex-grow flex-col overflow-hidden ">
+      <div className="flex flex-grow flex-col overflow-hidden pr-2">
         <div className="pb-5">
           <h1>{t('ranking-panel.title')}</h1>
-          <h2 className="text-sm">{t('ranking-panel.subtitle')}</h2>
+          <p className="text-sm">{<Trans i18nKey="ranking-panel.subtitle" />}</p>
```

Comment:
FYI this little change increased the bundle size by 7.3 kb and that the subtitle is constantly re-rendering on a datetime change.

I am fixing it in: #7647